utils/splTokens.ts

```diff
@@ -11,6 +11,7 @@ export type SplTokenInformation = {
 
 export type SupportedSplTokenNames =
   | 'USDC'
+  | 'UXD'
```

Comment:
iirc you did something to refacto token lists?

Reply:
tbh I don't really remember what's been done and what was planned to be done (but isn't yet)